backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -3,10 +3,13 @@
 
 import { bibiBloxberg } from '@/seeds/users/bibi-bloxberg'
 import { createContribution } from '@/seeds/graphql/mutations'
-import { login } from '@/seeds/graphql/queries'
+import { listContributions, login } from '@/seeds/graphql/queries'
 import { cleanDB, resetToken, testEnvironment } from '@test/helpers'
 import { GraphQLError } from 'graphql'
 import { userFactory } from '@/seeds/factory/user'
+import { creationFactory } from '@/seeds/factory/creation'
+import { creations } from '@/seeds/creation/index'
```

Comment:
These creations are admin creations. They are not suitable to test the user contributions.